eng/pipelines/common/templates/runtimes/send-to-helix-step.yml

```diff
@@ -26,60 +26,16 @@ parameters:
     ${{ if ne(parameters.condition, '') }}:
       condition: ${{ parameters.condition }}
-
-  - powershell: eng\common\msbuild.ps1 -ci ${{ parameters.helixProjectArguments }} /maxcpucount /bl:$(Build.SourcesDirectory)/artifacts/log/SendToHelix.binlog
-    displayName: ${{ parameters.displayName }}
-    ${{ if ne(parameters.condition, '') }}:
-      condition: ${{ parameters.condition }}
-    env:
-      __BuildArch: ${{ parameters.archType }}
-      __TargetOS: ${{ parameters.osGroup }}${{ parameters.osSubgroup }}
-      __BuildType: ${{ parameters.buildConfig }}
-      _Creator: ${{ parameters.creator }}
-      _PublishTestResults: ${{ parameters.publishTestResults }}
-      _HelixAccessToken: ${{ parameters.helixAccessToken }}
-      _HelixBuild: ${{ parameters.helixBuild }}
-      _HelixSource: ${{ parameters.helixSource }}
-      _HelixTargetQueues: ${{ join(',', parameters.helixQueues) }}
-      _HelixType: ${{ parameters.helixType }}
-      _RunCrossGen: ${{ parameters.runCrossGen }}
-      _RunCrossGen2: ${{ parameters.runCrossGen2 }}
-      _CompositeBuildMode: ${{ parameters.compositeBuildMode }}
-      _RunInUnloadableContext: ${{ parameters.runInUnloadableContext }}
-      _LongRunningGcTests: ${{ parameters.longRunningGcTests }}
-      _GcSimulatorTests: ${{ parameters.gcSimulatorTests }}
-      _Scenarios: ${{ join(',', parameters.scenarios) }}
-      _TimeoutPerTestCollectionInMinutes: ${{ parameters.timeoutPerTestCollectionInMinutes }}
-      _TimeoutPerTestInMinutes: ${{ parameters.timeoutPerTestInMinutes }}
-      runtimeFlavorDisplayName: ${{ parameters.runtimeFlavorDisplayName }}
-      _RuntimeVariant: ${{ parameters.runtimeVariant }}
-      ${{ if eq(parameters.publishTestResults, 'true') }}:
-        SYSTEM_ACCESSTOKEN: $(System.AccessToken)
-      # TODO: remove NUGET_PACKAGES once https://github.com/dotnet/arcade/issues/1578 is fixed
-      NUGET_PACKAGES: $(Build.SourcesDirectory)\.packages 
-
-- ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
-  # TODO: Remove and consolidate this when we move to arcade via init-tools.sh.
-  - script: $(Build.SourcesDirectory)/eng/common/build.sh /p:DotNetPublishToBlobFeed=true --ci --restore --projects $(Build.SourcesDirectory)/eng/empty.csproj
-    displayName: Restore blob feed tasks
-    ${{ if ne(parameters.condition, '') }}:
-      condition: ${{ parameters.condition }}
-    ${{ if eq(parameters.osGroup, 'FreeBSD') }}:
-      env:
-        # Arcade uses this SDK instead of trying to restore one.
-        DotNetCoreSdkDir: /usr/local/dotnet
-
-  - script: $(Build.SourcesDirectory)/eng/common/msbuild.sh --ci ${{ parameters.helixProjectArguments }} /maxcpucount /bl:$(Build.SourcesDirectory)/artifacts/log/SendToHelix.binlog
-    displayName: ${{ parameters.displayName }}
-    ${{ if ne(parameters.condition, '') }}:
-      condition: ${{ parameters.condition }}
-    env:
+    ${{ if eq(parameters.condition, '') }}:
```

Comment:
you could remove this and make the default condition value succeeded().

Reply:
I like the suggestion but I'm not sure if I understand it properly - are you saying that I should put <code>condition: succeeded()</code> in the parameters section? We're not using such construct anywhere in our scripts, is that going to work? Or did you mean just putting <code>condition: true</code> there and using the <code>succeeded()</code> function in construction of the "final" condition like we do in several scripts, something like <code>condition: and(succeeded(), ${{ parameters.condition }})</code>?

Reply:
I think we should do: `condition: true` in the parameters section and then use it like: `condition: and(succeeded(), ${{ parameters.condition }})`

Reply:
Thanks, fixed.